eyes_core/applitools/core/batch_close.py

```diff
@@ -25,6 +27,11 @@ def set_api_key(self, api_key):
         self.api_key = api_key
         return self
 
+    def set_proxy(self, proxy):
+        # type: (ProxySettings) -> _EnabledBatchClose
+        self.proxy = proxy
```

Comment:
We could add `argument_guard.is_a(proxy, ProxySettings)` to verify correctness

Reply:
Done!